Action.c

```diff
@@ -410,6 +410,25 @@ static Htop_Reaction actionStrace(State* st) {
    return HTOP_REFRESH | HTOP_REDRAW_BAR;
 }
 
+static Htop_Reaction actionDebugger(State* st) {
+   Settings* settings = st->settings;
+   Process* p = (Process*) Panel_getSelected(st->panel);
+   if (!p) return HTOP_OK;
+
+   char buf[32];
+   xSnprintf(buf, sizeof(buf), "%s -p %d", settings->debuggerTool, (int) p->pid);
+
+   endwin();
+   int result = system(buf);
+   refresh();
+
+   // we don't really care about the invocation result
+   (void) result;
+
+   CRT_enableDelay();
+   return HTOP_REFRESH | HTOP_REDRAW_BAR;
```

Comment:
Since `endwin()` get called, it's probably more cautious to include `HTOP_UPDATE_PANELHDR`